locales/en/translations.json

```diff
@@ -171,8 +171,13 @@
                     "title": "What is the limited partner's usual residential address",
                     "publicInformationLine1": "We will not show the limited partner's usual residential address on the public register.",
                     "postcodeOptional": "Postcode (optional)"
+                },
+                "principalOfficeAddress":{
+                    "title": "What is the limited partner's principal address",
```

Comment:
the general partner has the word 'office' in the title, but this one doesn't have it:
What is the general partner's principal office address?",

Reply:
Done.